app/views/directives/deploy-image.html

```diff
@@ -73,7 +73,8 @@
         administrator.
       </div>
       <h2>
-        {{app.name}}<span ng-if="import.tag">:{{import.tag}}</span>
+        <span ng-if="mode === 'dockerImage'">{{app.name}}<span ng-if="import.tag">:{{import.tag}}</span></span>
```

Comment:
Do we have a line that tells you we will create an image stream?  That seems like the right thing vs changing the name of the image that we loaded.